packages/__examples__/.storybook/stories/generateComponentExamples.js

```diff
@@ -132,12 +116,14 @@ module.exports = function generateComponentExamples(
 
   const maybeAddExample = (props) => {
```

Comment:
This method was the culprit for the bad performance. Its called >5000 times for some components and JSON.stringify is slow for such numbers.
Now it exits early if something is filtered, and the Object.entries() method is way faster.